dev-requirements.txt

```diff
@@ -3,13 +3,19 @@ apache-libcloud==0.17.0
 awscli==1.7.35
 backports.ssl-match-hostname==3.4.0.2
 botocore==1.0.0rc1
+cached-property==1.2.0
+chardet==2.3.0
 colorama==0.3.3
+coloredlogs==1.0.1
+deb-pkg-tools==1.34
```

Comment:
I'm guessing all the other additions to this file are dependencies of `deb-pkg-tools`?